src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -117,12 +117,16 @@ public int getMetaFromState(@NotNull IBlockState state)
         return state.getValue(FACING).getIndex();
     }
 
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings ("deprecation")
     @Override
     public boolean isOpaqueCube(final IBlockState state)
     {
         return false;
     }
 
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)